### ISSUE TITLE
Still in use?

### ISSUE THREAD

**Issue by 1977er** [freifunkh/mesh_branchfilter#1]
@AiyionPrime Is this stuff still in use? Can this repo be archived?

**Comment by AiyionPrime** [comment]
Yes. Usually for events or demonstrations for new communities. Not unless it saves money.
